src/graphql/mutation/createInsertMutationField.js

```diff
@@ -87,7 +87,14 @@ const createPayloadType = table =>
           node: output,
         }),
       },
-
+      // Add foreign key field references.
+      ...fromPairs(
+        table.getForeignKeys().map(foreignKey => {
```

Comment:
@tim-field you can get a hold of me on gitter most of the time. Just oben a direct message with me. Hope I can help.